fem/tmop.cpp

```diff
@@ -1433,6 +1727,89 @@ void TMOP_Integrator::ComputeNormalizationEnergies(const GridFunction &x,
    }
 }
 
+void TMOP_Integrator::ComputeMinJac(const Vector &x,
+                                    const FiniteElementSpace &fes)
+{
+   const IntegrationRule *ir = IntRule;
+   if (!ir)
+   {
+      ir = &(IntRules.Get(fes.GetFE(0)->GetGeomType(),
+                          2*fes.GetFE(0)->GetOrder() + 3)); // <---
```

Comment:
These integration orders are always suspicious :-)  It's nice they are marked with `<--` all over MFEM but maybe we should think of some other solution :-)